tests/integration/pilot/envoyfilter/main_test.go

```diff
@@ -78,7 +79,6 @@ spec:
     match:
       context: SIDECAR_INBOUND
       listener:
-        portNumber: 80
```

Comment:
@rshriram I had to remove this due to this type of matching no longer working with the single inbound listener